src/overlay/OverlayManagerImpl.cpp

```diff
@@ -113,7 +114,8 @@ OverlayManagerImpl::PeersList::removePeer(Peer* peer)
     if (pendingIt != std::end(mPending))
     {
         CLOG(DEBUG, "Overlay") << "Dropping pending " << mDirectionString
-                               << " peer: " << peer->toString();
+                               << " peer: " << peer->toString() << " @"
```

Comment:
Did you intend for no space between "@" and the port? This is the case in every log line.

Reply:
yes, makes it simpler to grep